reference/checkouts.v3.yml

```
@@ -346,36 +344,28 @@ paths:
           schema:
             $ref: '#/definitions/UpdateConsignmentRequest'
           x-examples:
-            application/json: |-
-              {
-                "shipping_address": {
-                  "email": "jane2@example.com",
-                  "country_code": "US",
-                  "first_name": "Testing",
-                  "last_name": "BigCommerce",
-                  "company": "BigCommerce",
-                  "address1": "1234 Main Street",
-                  "address2": "1234",
-                  "city": "Austin",
-                  "state_or_province": "Texas",
-                  "state_or_province_code": "TX",
-                  "postal_code": "78751",
-                  "phone": "+481526687548x4{1-5}",
-                  "custom_fields": [
-                    {
-                      "field_id": "field_25",
-                      "field_value": "You're Welcome"
-                    },
-                  ]
-                },
-                "line_items": [
-                  {
-                    "item_id": "15429582-222e-499e-8d22-d2930a0cfc0e",
-                    "quantity": 1
-                  },
-                ],
-                "shipping_option_id": "156fb123-68e1-40a2-8a48-98ceb04031c1"
-              }
+            Updating shipping optionID:
+              shipping_option_id: 9241669174884c2f2e83b3adabf03f83
+            Updating shipping address:
+              shipping_address:
+                email: jane2@example.com
+                country_code: US
+                first_name: Testing
+                last_name: BigCommerce
+                company: BigCommerce
+                address1: 1234 Main Street
```

this starts with a number, so force a string with quotes
```suggestion
                address1: '1234 Main Street'
```